### ISSUE TITLE
developer file: OKAY to use "docker-compose pull" statement?

### ISSUE THREAD

**Issue by dwk2** [Islandora-Collaboration-Group/ISLE-Documentation#50]
In this developer file `docs\05_developer_docs\dev_build_guide.md`, may we replace the multiple docker pull statements with a single `docker-compose pull` statement?

Current documentation shows the following:

Step 10: Test the images pushed.
Pull down the newly pushed images.

docker pull islandoracollabgroup/isle-apache:latest
docker pull islandoracollabgroup/isle-fedora:latest
docker pull islandoracollabgroup/isle-mysql:latest
docker pull islandoracollabgroup/isle-proxy:latest
docker pull islandoracollabgroup/isle-solr:latest

**Comment by br2490** [comment]
Yes, please. Follow the quick guide's instructions for all intents from isle-dev repo.

**Comment by dwk2** [comment]
Thank you Ben. I created a PR to update the obvious changes. 

Closing this ticket.
